ocspd/core/certmodel.py

```diff
@@ -103,6 +103,11 @@ def recycle_staple(self, minimum_validity):
             )
             return False
 
+        # from haproxy docs: [...] The content of this file is optional [...]
+        if len(staple) == 0:
```

Comment:
`staple` is not yet defined at this point, did you intend to add this at line ~120?